tst/EnergyPlus/unit/AirflowNetworkBalanceManager.unit.cc

```diff
@@ -7940,6 +7961,685 @@ namespace EnergyPlus {
 		EXPECT_NEAR( cp105S*0.5*118.41123742118911, DataAirflowNetwork::AirflowNetworkNodeSimu( 5 ).PZ, 1e-13 );
 	}
 
+	TEST_F( EnergyPlusFixture, TestExternalNodesWithLocalAirNode ) {
```

Comment:
Looks like this unit test isn't completely implemented yet, but this is definitely what we want to see.

Reply:
I modified it a bit. Any hint on what else is missing? Thanks.

Reply:
Unless I've missed it, the unit test doesn't appear to test the creation of the outdoor air nodes. That's really what I'd like to see, even if it is only one.